CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Additionally, default label `span_status` is renamed to `status_code`.
 * [BUGFIX] Fix nil pointer panic when the trace by id path errors. [#1441](https://github.com/grafana/tempo/pull/1441) (@joe-elliott)
 * [BUGFIX] Update tempo microservices Helm values example which missed the 'enabled' key for thriftHttp. [#1472](https://github.com/grafana/tempo/pull/1472) (@hajowieland)
 * [BUGFIX] Fix race condition in forwarder overrides loop. [1468](https://github.com/grafana/tempo/pull/1468) (@mapno)
+* [BUGFIX] Fix bug with FindTraceByID logic due to lexicographic ordering of TraceID string. Migrate TraceID column in parquet schema to byte slices. [#1531](https://github.com/grafana/tempo/pull/1531) (@annanay25)
```

Comment:
Until parquet is in a release I don't think we should update the changelog for all the bugfixes/improvements to get it ready. Open to other opinions here.

Reply:
Appended to the parquet feature in the CHANGELOG - let me know if this looks ok